packages/teleport/src/Kubes/ButtonView.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+Copyright 2021 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+import React from 'react';
+import { ButtonPrimary } from 'design';
+
+const docUrl =
+  'https://goteleport.com/docs/kubernetes-access/guides/multiple-clusters/';
+
+export default function ButtonView(props: Props) {
+  const { isEnterprise, isLeafCluster, canCreate } = props;
+
+  if (isEnterprise) {
+    if (isLeafCluster || !canCreate) {
+      return null;
+    }
+  }
```

Comment:
i don't think its necessary to create another file for this, what we can do instead is create a flag:

```
const showButton = !isEnterprise || (!isLeafCluster && canCreate)
```

First condition says OSS always shows button, b/c !isEnterprise == isOSS
Second condition says, show button if enterprise ( b/c first condition failed) and not a leaf cluster, and can create.